### PR TITLE
Draft for package walkthrough

### LINKED ISSUE #45: Documentation: My first package walkthrough
Ensure documentation is ready and clear to new contributors by adding a `My first package walkthrough` section.

Related to https://github.com/mandiant/VM-Packages/issues/28 and https://github.com/mandiant/VM-Packages/issues/27.

### PR DESCRIPTION
**Pull request by mr-tz**
Using this to allow for reviews. Once finalized this will go into the wiki.

closes #45

### CODE REVIEW

File: doc-first-package.md
Comment (Ana06) [inline doc-first-package.md:56]: creating a new branch is not really necessary.

File: doc-first-package.md
Comment (Ana06) [inline doc-first-package.md:48]: I would mention here also the documentation: https://github.com/mandiant/VM-Packages/wiki/Contributing

File: doc-first-package.md
Comment (Ana06) [inline doc-first-package.md:5]: I think it would be good to add some screenshots 🤔

File: doc-first-package.md
Comment (Ana06) [inline doc-first-package.md:6]: What about making this a link?

File: doc-first-package.md
Comment (Ana06) [inline doc-first-package.md:14]: what about linking Github documentation about how to do it? https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally

I think we should add that they may need to open a new PR because I think only maintainers can push to a PR pen by someone else.

**Review by Ana06 — COMMENTED** [review]
This great @mr-tz! Thanks for writing this.

I am missing some information about when/why I would want to create a package manually or modify an automatically created PR. I think we should mention all kind of cool things you can do and mention few examples.

File: doc-first-package.md
Comment (mr-tz) [reply doc-first-package.md:61]: added

File: doc-first-package.md
Comment (Ana06) [inline doc-first-package.md:70]: maybe mention that linter/tests also run in the PR and that the PR can only be approved if they succeed.

**Review by Ana06 — APPROVED** [review]
Remember to squash [address feedback](https://github.com/mandiant/VM-Packages/pull/95/commits/f32cca3fe3edb290899a8538510a1b470150b409) (either manually or when merging ) 😄

**Comment by mr-tz** [comment]
I'll copy this to the Wiki or do you think we should add the file itself?

**Comment by Ana06** [comment]
I am fine with wiki. We should modify the readme to link to the most important documentation.

File: doc-first-package.md
Comment (mr-tz) [reply doc-first-package.md:14]: "we should add people to use urls that are not willing to change and include the version. then less things break"

**Pull request closed without merging**
